src/Razor/Microsoft.NET.Sdk.Razor/src/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Component.targets

```diff
@@ -231,6 +232,7 @@ Copyright (c) .NET Foundation. All rights reserved.
          ResponseFiles="$(CompilerResponseFile)"
          RuntimeMetadataVersion="$(RuntimeMetadataVersion)"
          SharedCompilationId="$(SharedCompilationId)"
+         SkipAnalyzers="true"
```

Comment:
Note: this will only work once they are using the RC1 SDK correct? This change in our targets didn't make P8 IIRC

Reply:
I'm not 100% sure, but I just grabbed the preview8 SDK bits and it seems to be there in the targets.

Reply:
Just verified myself and yes it made .NET P8.